lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -572,7 +572,7 @@ def main():
         http_timeout=dict(type='int'),
         load_path=dict(type='path'),
         name=dict(type='str', required=True),
-        nocache=dict(type='str', default=False),
+        nocache=dict(type='bool', default=False),
```

Comment:
I'm not sure what needs to be updated in the docs, I believe this PR fixes the behavior to fit what's there?